iceoryx2-bb/log/src/lib.rs

```diff
@@ -10,6 +10,10 @@
 //
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+#![warn(clippy::alloc_instead_of_core)]
```

Comment:
We should do it the other way around. We should enable this globally in our clippy rules and ignore this warning for CLI, examples and benchmarks. But even there it is not a problem

Reply:
This applies to all `lib.rs` files.

Reply:
I tried, but was not able to configure it via `clippy.toml`